include/glow/Graph/Node.h

```diff
@@ -207,7 +207,7 @@ struct NodeUse {
   /// \returns the instruction that the use refers to.
   NodeHandle *get() const { return site_; }
   /// Get the node containing this use.
-  Node *getUser() { return site_->parent_; }
+  Node *getUser() const { return site_->parent_; }
```

Comment:
could this return const Node *? It's a bit tricky to expose the internals of the const object and one could modify the const object this way.

Reply:
@rdzhabarov   We could. I wonder though what happens if you want to get a user of a node and then mutate that user. If we make `getUser` const, then that would require an explicit cast. Or we can have two `getUser` methods for const and non-const uses. What do you prefer?

Reply:
I added two versions of `getUser`, one that is const returning  `const Node *` and another which is non-const.